torch/csrc/jit/ir/scope.h

```diff
@@ -112,8 +111,12 @@ struct TORCH_API InlinedCallStack : public c10::intrusive_ptr_target {
   SourceRange source_range_;
   InlinedCallStackPtr intrusive_from_this();
   c10::optional<ModuleInstanceInfo> module_instance_info_;
+  std::string modulePath_;
+  bool initialized_ = true;
```

Comment:
Q. Where are you using the `initialized_` variable?

Reply:
`InlinedCallStack ` is an optional field. If a node did not come from any inlined function, its callstack will be empty and modulePath_ can't be set.I was thinking to use initialized field to indicate its validity, instead of using the x.has_value() from std::optional. Anyway, I will take @kimishpatel suggestion and use std::unordered_map, such that we don't need to add extra field in node.